vignettes/Introduction.Rmd

```diff
@@ -7,11 +7,11 @@ library(pryr)
 knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
 ```
 
-The R6 package provides an implementation of encapsulated object-oriented programming for R (also sometimes referred to as classical object-oriented programming). It is similar to R's reference classes, but it is more efficient and doesn't depend on S4 classes and the methods package.
+The R6 package provides an implementation of encapsulated object-oriented programming for R (also sometimes referred to as classical object-oriented programming). It is similar to R's reference classes, but it is more efficient and doesn't depend on S4 classes and the *methods* package.
 
 ## R6 classes
 
-R6 classes are similar to R's reference classes, but are lighter weight, and avoid some issues that come along with using S4 (R's reference classes are based on S4). For more information about speed and memory footprint, see the Performance article.
+R6 classes are similar to R's reference classes, but are lighter weight, and avoid some issues that come along with using S4 (R's reference classes are based on S4). For more information about speed and memory footprint, see `vignette("Performance")`.
```

Comment:
The reason this refers to "the Performance article" is because users are most likely to see this on the website at https://r6.r-lib.org/articles/Introduction.html. Perhaps it could be converted to a link, like:

```md
[Performance article](Performance.html)
```

Reply:
But that is exactly why I made this change! This way the vignette article will be auto-linked.

As per pkgdown [docs](https://pkgdown.r-lib.org/articles/linking.html):

![image](https://user-images.githubusercontent.com/11330453/145082985-82c50690-830f-45ce-ac30-3190d2ff6dec.png)

Reply:
Ah, I was unaware of that feature of pkgdown. Sounds good.